libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts

```diff
@@ -54,7 +54,8 @@ export class SkyToggleSwitchComponent
   /**
    * The ARIA label for the toggle switch. This sets the toggle switch's `aria-label`
    * attribute to provide a text equivalent for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
-   * If the `sky-toggle-switch-label` component displays a visible label, do not use this property.
+   * Use a context-sensitive label, such as "Annual fundraiser is active." Context is especially important when multiple toggle switches are in close proximity.
```

Comment:
The "Annual fundraiser is active" only reflects one state for the toggle switch. So I think this ARIA label is incorrect half the time. I mean, if the toggle switch is in the unselected state but the ARIA label says "Annual fundraiser is active," I think that might be confusing to screen reader users.

Should we provide an example that works for both selected and unselected states? Or call out that both states need to be accounted for? I'm not sure how to do that without getting long-winded, but my first thought is to flesh this out to something like "Use a context-sensitive label. For example, when a toggle switch activates and deactivates an annual fundraiser, use "Annual fundraiser is active" or "Annual fundraiser is inactive" depending on the selected state." That's much longer, but I _think_ it's more accurate. Let me know if you want to coordinate further or if I need to pull in Matt Gregg.